vac/70/eth-secpm.md

```diff
@@ -283,7 +283,11 @@ These identifiers MUST be computed according to Section 5.2 of [RFC9420](https:/
 Each member of a group presents a credential that provides one or more identities for the member and associates them with the member's signing key. 
 The identities and signing key are verified by the Authentication Service in use for a group.
 
-Credentials MUST follow the specifications of section 5.3 of [RFC9420](https://datatracker.ietf.org/doc/rfc9420/).  
+Credentials MUST follow the specifications of section 5.3 of [RFC9420](https://datatracker.ietf.org/doc/rfc9420/).
+
+Below follows the flow diagram for the generation of credentials. 
+Users MUST generate key pairs by themselves.
+![credentials](https://github.com/vacp2p/rfc-index/assets/74050285/04011f8f-db01-4ae1-a720-1865d4c4d143)
```

Comment:
I think these images are meant to be uploaded to the rfc-index, can you do that? and reference them this way?
context:
waku-rln-relay rfc-index: https://github.com/vacp2p/rfc-index/blob/69f28534075d1486ebff0f292cd4e2fa80f6fd1b/waku/standards/core/17/rln-relay.md?plain=1#L62C1-L62C51
and the images go in the spec/images directory
https://github.com/vacp2p/rfc-index/tree/main/waku/standards/core/17/images